KronorComponents/Fallback/SwiftUIView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FallbackWaitingView.swift
+//
+//
+//  Created by lorenzo on 2023-04-18.
+//
+
+import SwiftUI
+
+struct FallbackWaitingView: View {
```

Comment:
Filename should be the same as Struct is named.

Reply:
oops, good catch!